mobile/src/screens/chat/components/badge/badge.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Text, View } from 'react-native';
```

Comment:
import Text and View from '#libs/components/components'

Reply:
Done